concat-utils.php

```diff
@@ -24,7 +24,7 @@ public static function realpath( $url, $site_url ) {
 		$url_path = parse_url( $url, PHP_URL_PATH );
 		$site_url_path = parse_url( $site_url, PHP_URL_PATH );
 		// To avoid partial matches; subdir install at `/wp` would match `/wp-includes`
-		$site_url_path = trailingslashit( $site_url_path );
+		$site_url_path = is_null( $site_url_path ) ? '/' : trailingslashit( $site_url_path );
```

Comment:
Can we just return an empty string instead of `/` since that's what was the return value in previous PHP versions?

Reply:
```suggestion
		$site_url_path = trailingslashit( $site_url_path ?? '' );
```